### PR TITLE
Added note for usage of common datasets

### PR DESCRIPTION
**Pull request by UmashankarTriforce**
The path to place the datasets for use with MetalHead is not clear. This PR aims to add a note entry in `README.md` to specify where to place the datasets. Useful for first time users

### CODE REVIEW

**Comment by dhairyagandhi96** [comment]
The Metalhead.jl API has a method to detect datasets. Linking to the appropriate documentation is more flexible imo. See https://github.com/FluxML/Metalhead.jl#working-with-common-datasets

**Comment by UmashankarTriforce** [comment]
Ahh I missed this one. I'll modify the readme to house this point

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Models with a `cuda` folder can be loaded with NVIDIA GPU support, if you have a
 using Pkg; Pkg.activate("cuda"); Pkg.instantiate()
 using CuArrays
 ```
-Note: Place your downloaded dataset where MetalHead is installed, typically in `/home/<user>/.julia/packages/MetalHead/fYeSU/src/datasets`
+**Note**: For working with common datasets, refer <a href = https://github.com/FluxML/Metalhead.jl#working-with-common-datasets> here </a>
```
[the remark this answers is not in the document]
Comment (UmashankarTriforce) [reply README.md]: May I know what the markdown format is?

[the remark this answers is not in the document]
Comment (dhairyagandhi96) [reply README.md]: You can refer to how links have been added in the rest of the document. In general, its similar to `[<text>](<link>)`

See: 
https://raw.githubusercontent.com/FluxML/model-zoo/master/README.md

[the remark this answers is not in the document]
Comment (UmashankarTriforce) [reply README.md]: Have a look now. I think this should be fine

**Comment by dhairyagandhi96** [comment]
Thanks for the patch! @UmashankarTriforce

**Pull request merged**